toolchain/sem_ir/typed_insts.h

```diff
@@ -335,8 +335,8 @@ struct NoOp {
   // This instruction doesn't produce a value, so has no type.
 };
 
-struct Parameter {
-  static constexpr auto Kind = InstKind::Parameter.Define("parameter");
+struct Param {
+  static constexpr auto Kind = InstKind::Param.Define("parameter");
```

Comment:
```suggestion
  static constexpr auto Kind = InstKind::Param.Define("param");
```

Reply:
This should probably change. It'll cause a lot of test edits, but I don't think those need to be double-checked.

Reply:
It doesn't cause any test edits.